isort/settings.py

```diff
@@ -251,7 +251,7 @@ def __post_init__(self) -> None:
             if sys.version_info.major == 2 and sys.version_info.minor <= 6:
                 py_version = "2"
             elif sys.version_info.major == 3 and (
-                sys.version_info.minor <= 5 or sys.version_info.minor >= 10
+                sys.version_info.minor <= 6 or sys.version_info.minor >= 11
```

Comment:
The edge conditions seem wrong here. We previously had

Python 2.6 → `py_version = "2"`
Python 2.7 → `py_version = "27"`
Python 3.5 → `py_version = "3"`
Python 3.6 → `py_version = "36"`
Python 3.7 → `py_version = "37"`
Python 3.8 → `py_version = "38"`
Python 3.9 → `py_version = "39"`
Python 3.10 → `py_version = "3"`
Python 3.11 → `py_version = "3"`

and we now have

Python 2.6 → `py_version = "2"`
Python 2.7 → `py_version = "27"`
Python 3.5 → `py_version = "3"`
**Python 3.6 → `py_version = "3"`**
Python 3.7 → `py_version = "37"`
Python 3.8 → `py_version = "38"`
Python 3.9 → `py_version = "39"`
**Python 3.10 → `py_version = "310"`**
Python 3.11 → `py_version = "3"`

There’s no change here for Python 3.11.

The `<=` and `>=` should be `<` and `>`.